acceptance/ui/features/support/application.rb

```diff
@@ -1,4 +1,5 @@
 require 'uri'
+require 'json'
```

Comment:
No, it was part of some refactoring that got left behind after a few code moves.  I'll remove it.